extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -47,6 +47,7 @@ tags: {  # <amp-video>
     name: "controls"
     value: ""
   }
+  attrs: { name: "controlslist" }
```

Comment:
Is this case sensitive? if so, should these be `controlsList`?

Reply:
is not as I learned from https://github.com/ampproject/amphtml/pull/10976

Reply:
huh nice!